rosdep/python.yaml

```diff
@@ -1230,15 +1230,21 @@ python-github-pip:
       packages: [PyGithub]
 python-gitlab:
   debian:
-    '*': [python-gitlab]
     jessie:
       pip:
         packages: [python-gitlab]
+    buster: [python-gitlab]
```

Comment:
there is now no rule for `stretch`.
Something like this should do the job (explicit rule for jessie and stretch, wildcard for buster and releases that will come after)
```
   debian:
     '*': [python-gitlab]
     jessie:
       pip:
         packages: [python-gitlab]
     stretch:
       pip:
         packages: [python-gitlab]
```

Reply:
oh, OK, so the wildcard is used for coming releases too...

Reply:
Yes, sorry if it was unclear. The main goal of introducing wildcards is to allow us to not have to add one new entry everytime a new OS distro comes out. So ideally we would have specific rules for anything that diverges from what is in the latest distro, and use the wildcard for the latest and upcoming distributions

Reply:
I had a misunderstanding for the wildcards, thank you for claryfing.